homeassistant/components/bluetooth/__init__.py

```diff
@@ -241,8 +244,15 @@ def __init__(
     async def async_setup(self) -> None:
         """Set up BT Discovery."""
         try:
+            # TODO : fallback to active scanner if passive fails
             self.scanner = HaBleakScanner(
-                scanning_mode=SCANNING_MODE_TO_BLEAK[self.scanning_mode]
+                scanning_mode=SCANNING_MODE_TO_BLEAK[self.scanning_mode],
+                bluez=BlueZArgs(
+                    or_patterns=[
+                        OrPattern(0, AdvertisementDataType.FLAGS, b"\x06"),
```

Comment:
It would be nice if the byte flags could be abstracted to some constants that we can import from the bleak backend. What do the bytes mean?

Reply:
It's a matcher that is effectively match everything since the underlying kernel apis require a matcher even if one isn't desired.